### PR TITLE
postgresql_privs: add procedure type support

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Fixes: https://github.com/ansible-collections/community.general/issues/1002
postgresql_privs: add procedure type support

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
`plugins/modules/database/postgresql/postgresql_privs.py`

### CODE REVIEW

**Comment by Andersson007** [comment]
ready_for_review

File: plugins/modules/database/postgresql/postgresql_privs.py [hunk 1]
```diff
@@ -42,31 +42,33 @@
...
       database objects of type I(type) in the schema specified via I(schema).
       (This also works with PostgreSQL < 9.0.) (C(ALL_IN_SCHEMA) is available
        for C(function) and C(partition table) since Ansible 2.8)
+    - C(procedure) is supported since PostgreSQL 11 and M(community.general) collection 1.3.0.
     - If I(type) is C(database), this parameter can be omitted, in which case
       privileges are set for the database specified via I(database).
-    - 'If I(type) is I(function), colons (":") in object names will be
-      replaced with commas (needed to specify function signatures, see examples)'
+    - 'If I(type) is I(function) or I(procedure), colons (":") in object names will be
+      replaced with commas (needed to specify signatures, see examples)'
```
Comment (tcraxs) [inline plugins/modules/database/postgresql/postgresql_privs.py:64]: Is there a need to quote this two lines?

**Review by tcraxs — APPROVED** [review]
LGTM, Good work @Andersson007

File: plugins/modules/database/postgresql/postgresql_privs.py (hunk 1 above)
Comment (Andersson007) [reply plugins/modules/database/postgresql/postgresql_privs.py]: @tcraxs good spot, fixed, thanks!

**Comment by Andersson007** [comment]
@tcraxs thanks for reviewing!:)

**Pull request merged**